plugins/database/src/db_plugin.cpp

```diff
@@ -590,9 +597,7 @@ irods::error _childIsValid(
     }
     else if ( strlen( parent ) != 0 ) {
         // If the resource already has a parent it cannot be added as a child of another one
-        std::stringstream ss;
-        ss << "Child resource \"" << resc_name << "\" already has a parent \"" << parent << "\"";
-        irods::log( LOG_NOTICE, ss.str() );
+        log_db::info(R"(Child resource "{}" already has a parent "{}")", resc_name, parent);
```

Comment:
- Add function name prefix to log message
- `"{}"` -> `[{}]`

Reply:
Done.